spec/features/admin_deletes_content_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "rails_helper"
+
+feature "User Creates Account", %q(
+As an administrator of the Code Buddy Website
+I want to be able to delete users, code resources, and reviews
+So that I can get rid of inappropriate content and users.
+
+Acceptance Criteria
+
+
+) do
+
+  let(:user) { FactoryGirl.create(:user) }
+  let(:admin) { FactoryGirl.create(:admin) }
+  let(:buddy) { FactoryGirl.create(:buddy) }
+
+  context "User is admin" do
+
+    before(:each) do
+      visit new_user_session_path
+      fill_in "user[email]", with: admin.email
+      fill_in "user[password]", with: admin.password
+      click_button "Log in"
+    end
+
+    scenario 'admin views all users' do
+      user.save
+      visit admin_users_path
+      expect(page).to have_content(user.display_name)
+    end
+
+    scenario 'admin deletes a user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.